spring-core/src/main/java/org/springframework/cglib/core/ReflectUtils.java

```diff
@@ -45,7 +45,7 @@
  * @version $Id: ReflectUtils.java,v 1.30 2009/01/11 19:47:49 herbyderby Exp $
  */
 @SuppressWarnings({"rawtypes", "unchecked"})
-public class ReflectUtils {
+public abstract class ReflectUtils {
```

Comment:
`ReflectUtils` is a type from CGLIB which is _repackaged_ into `spring-core` using [Jar Jar](https://github.com/shevek/jarjar).

As such, it is not from Spring, and we therefore do not make such modifications to CBLIB classes.

Thus, please remove this change.